File: parley/src/androidTest/java/nu/parley/android/data/model/MockAgent.java
package nu.parley.android.data.model;

import java.util.Random;

public final class MockAgent {

    private static int generateRandomId() {
        return new Random().nextInt();
    }

    private static Agent create(String name) {
        return new Agent(name);
    }

    public static Agent Webuildapps = create("Webuildapps");
}

File: parley/src/main/java/nu/parley/android/util/mock/MockAgent.kt
package nu.parley.android.util.mock

import nu.parley.android.data.model.Agent
import java.util.Random

internal object MockAgent {
    private fun generateRandomId() = Random().nextInt()

    private fun create(name: String): Agent {
        return Agent(name)
    }

    var Webuildapps: Agent = create("Webuildapps")
}
